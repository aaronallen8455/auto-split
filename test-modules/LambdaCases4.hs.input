main :: IO ()
main = pure ()

test :: Bool -> Maybe Bool -> Bool
test = \cases
  _ SPLIT -> False
