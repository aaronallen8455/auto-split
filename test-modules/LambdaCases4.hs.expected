main :: IO ()
main = pure ()

test :: Bool -> Maybe Bool -> Bool
test = \cases
  _ Nothing -> False
  _ (Just _) -> False
